save-plugins/warn-plugin/src/commonMain/kotlin/org/cqfn/save/plugin/warn/ResourceFormatException.kt

```diff
@@ -0,0 +1,8 @@
+package org.cqfn.save.plugin.warn
+
+import org.cqfn.save.core.plugin.PluginException
+
+/**
+ * An [Exception] that can be thrown when parsing a file
+ */
+class ResourceFormatException(message: String) : PluginException(message)
```

Comment:
This should be in `save-core` two, because all plugins operate with test resources and may throw it